server/app/services/rpc_server.rb

```diff
@@ -13,12 +13,12 @@ class RpcServer
   }
 
   class Error < StandardError
-    attr_accessor :code, :message, :backtrace
+    attr_reader :code
 
-    def initialize(code, message, backtrace = nil)
```

Comment:
The `RpcServer::Error` are only used locally, and do not need any `remote_backtrace`.